### PR TITLE
Remove Dir::chdir

### PR DESCRIPTION
**Pull request by rhysd**
`Dir::cd` should be an alias of `Dir::chdir`.  I added missing block version of `Dir::cd`.

### CODE REVIEW

**Comment by jhass** [comment]
Shouldn't we rather drop one of the two?

**Comment by rhysd** [comment]
I can't determine it because I don't know the policy of alias in Crystal standard library.

**Comment by asterite** [comment]
We prefer avoiding aliases. Maybe `cd` is better because it's shorter and known from the command line?

**Comment by rhysd** [comment]
I understood.  I also prefer `cd` to `chdir`.  I'll modify code and push it.

**Comment by rhysd** [comment]
Ah, tests fail only in Linux... I don't have Linux 64bit environment currently.  Does anyone know why tests fail in Linux?

File: spec/std/dir_spec.cr [hunk 1]
```diff
@@ -131,25 +131,23 @@ describe "Dir" do
...
 
-  describe "chdir" do
+  describe "cd" do
     it "should work" do
       cwd = Dir.working_directory
-      Dir.chdir("..")
+      Dir.cd("..")
       Dir.working_directory.should_not eq(cwd)
-      Dir.cd(cwd)
-      Dir.working_directory.should eq(cwd)
```
Comment (jhass) [inline spec/std/dir_spec.cr]: Why did you remove these two lines?

Comment (rhysd) [inline spec/std/dir_spec.cr]: Ah, I mistook... I'll fix.  Thanks!

**Comment by rhysd** [comment]
Test passed! Thank you @jhass.

**Comment by asterite** [comment]
Thank you!
